client/components/UserDecalPanel.js

```diff
@@ -87,6 +90,37 @@ class UserDetailsPanel extends Component {
     return null
   }
 
+  _renderDecalCode = (decal, index) => (
+    <>
+      <div className="decal-code" key={`code${index}`}>
+        <button
+          key={`toggle${index}`}
+          aria-label="toggle-decal"
```

Comment:
aria-label should be human readable explanatory text, as it is read by screen readers.
```suggestion
          aria-label="Show decal code"
```